### PR TITLE
fix jquery objects being serialized

### PR DESCRIPTION
**Pull request by bradencanderson**

### CODE REVIEW

File: panelSrc/components/PinspectorApp.js [hunk 1]
```diff
@@ -65,12 +65,18 @@ chrome.devtools.inspectedWindow.eval(`
...
         var whitelist = ['cid', 'data', 'options', 'resource', 'extraData', 'extraState'];
-        return Object.keys(module).reduce(function(obj, key) {
+        var mod = Object.keys(module).reduce(function(obj, key) {
             if (whitelist.indexOf(key) !== -1) {
                 obj[key] = module[key];
             }
             return obj;
         }, { name: module.className, children: children });
+        return JSON.parse(JSON.stringify(mod, function(property, value) {
+            if (value instanceof jQuery) {
```
Comment (zackargyle) [inline panelSrc/components/PinspectorApp.js]: Any reason we can't do this check in the reduce above? Seems unnecessary to parse/stringify.

Comment (bradencanderson) [inline panelSrc/components/PinspectorApp.js]: `JSON.stringify` does a deep traversal by default, whereas the reduce above is a shallow traversal. The errors we're getting are in a submodule's options.

Comment (bradencanderson) [inline panelSrc/components/PinspectorApp.js]: It might be a good idea to just move all of our whitelist logic to JSON.stringify.

**Comment by zackargyle** [comment]
Alright sounds good. Didn't think about the deep traversal aspect.

**Pull request merged**